app/src/App.scss

```diff
@@ -33,8 +33,8 @@ body {
 }
 
 .app-container {
-	width: 100vw;
-	height: 100vh;
+	width: 100%;
```

Comment:
Is this change related to the escape key fix? The view height/width measurement is not the same thing as a percent of the parent element.